lib/msal-core/src/UserAgentApplication.ts

```diff
@@ -2205,13 +2166,15 @@ export class UserAgentApplication {
   */
   // TODO: Check if this can be combined with validateInputScope()
   private filterScopes(scopes: Array<string>): Array<string> {
```

Comment:
looks like we remove them here, and add it to every request before we send to server. Ideally we just add if needed, and the code seem to be doing that already, so not sure the value add of this method. We may be able to remove this, but if you are not sure, leave it as is

Reply:
removed the function as it seems like we don't use it anywhere else. We already filter before we send to the server.